.prerequisiteversions

```diff
@@ -0,0 +1 @@
+CMake=3.7.2
```

Comment:
To talk a little about impact: this means that when we introduce bootstrap.sh/ps1, we'll need to either rename this file to distinguish against nuget packages, or add a workaround (maybe just ignoring `CMake` by name). That depends on the order we plan on finishing this work in, though: I'm imagining we're aiming for (1) CMake (2) bootstrap (3) unify tools/prereqs.